handler.go

```diff
@@ -843,6 +844,12 @@ func (h *Handler) readProtobufQueryRequest(r *http.Request) (*QueryRequest, erro
 // readURLQueryRequest parses query parameters from URL parameters from r.
 func (h *Handler) readURLQueryRequest(r *http.Request) (*QueryRequest, error) {
 	q := r.URL.Query()
+	validQuery := h.getValidURLQuery(r)
```

Comment:
Since the value of `validQuery` is always the same, would it make sense to save it in a global variable ?